identity-provider/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,171 @@
+     setOpen(true);
+   };
+ 
+   const handleClose = () => {
+     setOpen(false);
+   };
+ // *********************************************
+
+//  We want dialog to show up for each button, so do we need to put the return inside the buttons or keep it in the function?
+    const editInfoButton = () => {
+        console.log('This button will bring up a typed out alert which shows your information and contains x butttons to delete info with a cancel and save button.')
+    }
+
+    const unlinkButton = () => {
+        console.log('This button will bring up a typed alert which will ask you if you want to unlink from the organization the user pressed, with a cancel and confirm button.')
+    }
+    return (
+        <>
+        <h1>Your Volunteer Organizations</h1>
+        <h2>Select an account to sign in </h2>
+        <a href = 'https://www.redcross.org/'></a>
+        <button type = 'submit' onClick = {editInfoButton}> View Info</button>
+        {/* We need a patch request for the unlink button to update the list of organizations once the user presses 'confirm' */}
+        <button type = 'submit' onClick = {unlinkButton}>Unlink</button>
+
+        {/* This Dialog will come up after you click the "Edit Info" button. */}
+
+        <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby={dialogTitle}
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">
+          {"Your Information"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            {/* Placeholder unordered list for the list of organizations: Will want to show what user's information is linked to those organizations*/}
+            <p> You are currently sharing this information with ______</p>
+          <ul>
+             <li>Username</li>
+             <li>Name</li>
+             <li>Address</li>
+             </ul>
+             {/* ******************************************* */}
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <Button onClick={handleClose}>Cancel</Button>
+          <Button onClick={handleClose} autoFocus>
+            Save
+          </Button>
+        </DialogActions>
+      </Dialog>
+
+      {/* This Dialog will come up after you click the "Unlink" button. */}
+
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">
+          {"Are you sure you would like to unlink your account from ______?"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <Button onClick={handleClose}>Cancel</Button>
+          <Button onClick={handleClose} autoFocus>
```

Comment:
This would instead call a brand new function you need to create called something like "confirmUnlink" that uses the currentOrganization state variable to send an unlink API request